### ISSUE TITLE
bug: Use of worker-farm breaks jest's --runInBand & --max-concurrent-workers options

### ISSUE THREAD

**Issue by hulkish** [rogeliog/jest-runner-mocha#16]
Usage of `worker-farm` in a jest runner:
- breaks jest's `--runInBand` option
- breaks jest's `--max-concurrent-workers` option
- `jest-worker` is already handling the concurrency behavior
- the spawning of child.processes will perplex between `jest-worker` and `worker-farm`. 
- this actually makes it very dangerous for CI environments

Instead, i recommend:
- dropping usage of `worker-farm` all together  
- use [the same approach](https://github.com/jest-community/jest-runner-eslint/blob/master/src/index.js) which `jest-runner-eslint` is using

**Comment by rogeliog** [comment]
I agree, we should just migrate to use create-jest-runner cc: @SimenB

**Comment by SimenB** [comment]
Yup yup! Really straight forward to use 🙂
